crates/red_knot_python_semantic/resources/mdtest/comparison/intersections.md

```diff
@@ -14,7 +14,12 @@ class Child1(Base):
     def __eq__(self, other) -> Literal[True]:
         return True
 
-class Child2(Base): ...
+    def __gt__(self, other) -> bool:
```

Comment:
@sharkdp I'm pretty sure I now changed the test in a way that it is no longer testing what it should but I'm not sure how to fix it without breaking the test itself.

Reply:
> I think you should put this method on `Base` instead of `Child1` and `Child2`.

Yes. Reverting to the state on main and adding `__get__` to `Base` keeps the original intention of the test and fixes the error. Thanks!